pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -34,6 +36,8 @@ def __init__(
         date_column: str,
         channel_columns: List[str],
         adstock_max_lag: int,
+        time_varying_media_effect: bool = False,
+        time_varying_intercept: bool = False,
```

Comment:
I think it would be great if we could control the HSGP. Could we add an `hsgp_config` similar to how we defined the priors? So the more "expert" user could play with how much flexibility they want to give to the HSGP?

Reply:
Maybe better to just support that in the model config? The chief thing you'd want to control is the covariance function, m and L. But for MMM this can even be simplified to just lengthscale and m, I believe since optimal L can be calculated form the length of the data and the lengthscale.